localisation/00_wc_artifact_localisation.csv

```diff
@@ -19,7 +19,7 @@ doomhammer_desc;Forged on Draenor in a lava basin and passed down through genera
 eye_of_sargeras;Eye of Sargeras;;Auge von Sargeras;;;;;;;;;;;x
 eye_of_sargeras_desc;The Eye of Sargeras is a powerful gem-like artifact previously used by the Avatar of Sargeras. This magical stone thrums with demonic energies.;;Das Auge des Sargeras ist ein m�chtiges, edelstein�hnliches Artefakt, das zuvor vom Avatar des Sargeras benutzt wurde. Dieser magische Stein pulsiert mit d�monischen Energien.;;;;;;;;;;;x
 felomelorn;Felo'melorn;;Felo'melorn;;;;;;;;;;;x
-felomelorn_desc;This runeblade, known also as Flamestrike, is a family relic of the Sunstrider dynasty. Anastarian Sunstrider wielded it against the Amani trolls after the founding of Quel'thalas, having originally inherited it from Dath'Remar.;;Diese Runenklinge, auch bekannt als Flammensto�, ist ein Familienrelikt der Sunstrider-Dynastie. Anastarian Sunstrider f�hrte sie nach der Gr�ndung von Quel'thalas gegen die Amani-Trolle, nachdem er sie urspr�nglich von Dath'Remar geerbt hatte.;;;;;;;;;;;x
+felomelorn_desc;This runeblade, known also as Flamestrike, is a family relic of the Sunstrider dynasty. Anasterian Sunstrider wielded it against the Amani trolls after the founding of Quel'Thalas, having originally inherited it from Dath'Remar.;;Diese Runenklinge, auch bekannt als Flammensto�, ist ein Familienrelikt der Sunstrider-Dynastie. Anasterian Sunstrider f�hrte sie nach der Gr�ndung von Quel'Thalas gegen die Amani-Trolle, nachdem er sie von Dath'Remar geerbt hatte.;;;;;;;;;;;x
```

Comment:
```suggestion
felomelorn_desc;This runeblade, known also as Flamestrike, is a family relic of the Sunstrider dynasty. Anasterian Sunstrider wielded it against the Amani trolls after the founding of Quel'Thalas, having originally inherited it from Dath'Remar Sunstrider, the founder of Quel'Thalas.;;Diese Runenklinge, auch bekannt als Flammensto�, ist ein Familienrelikt der Sunstrider-Dynastie. Anasterian Sunstrider f�hrte sie nach der Gr�ndung von Quel'Thalas gegen die Amani-Trolle, nachdem er sie von Dath'Remar geerbt hatte.;;;;;;;;;;;x
```

Reply:
This was shortened on purpose so it fits into description box with bigger interface mod.